lib/rouge/lexers/make.rb

```diff
@@ -96,7 +96,7 @@ def initialize(opts={})
         end
 
         rule(/\\./m) { delegate @shell }
-        stop = /\$\(|\(|\)|\n|\\/
+        stop = /\$\(|\(|\)|\n|\\|$/
```

Comment:
can we drop the `\n` from this regex? I think it should be delegated to the shell lexer anyways.